src/cata_tiles.cpp

```diff
@@ -806,8 +822,8 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
 bool cata_tiles::draw_tile_at(tile_type *tile, int x, int y, int rota)
 {
     // don't need to check for tile existance, should always exist if it gets this far
-    const int fg = tile->fg;
-    const int bg = tile->bg;
+    const std::vector<int> fg = tile->fg;
```

Comment:
The copy of a single int (previous type of fg, bg) is fine here, however copying the vector is not good, use a const reference instead.